### ISSUE TITLE
Undo / Redo feature

### ISSUE THREAD

**Issue by dmarcos** [aframevr/aframe-inspector#21]
I removed a component by mistake and I cried.

**Comment by caseyyee** [comment]
Also just moving a component.  +1, this is much needed.

**Comment by iburguera** [comment]
Is this enhancement added to the last version of a-frame inspector? Undo or ctrl+z feature is something useful when moving objects by mistake and want to come back.

**Comment by dmarcos** [comment]
No. If hasn't been added yet. Not a top priority at the moment.

**Comment by rmjgtking** [comment]
Oh This is really bad. It should be available to undo

**Comment by jackmurraybrown** [comment]
What would be a possible way to implement this? Where is the best place to start? Really needing this feature.

**Comment by vincentfretin** [comment]
https://threejs.org/editor/ has an undo/redo feature.
Main implementation in this file https://github.com/mrdoob/three.js/blob/master/editor/js/History.js
each action is a command
https://github.com/mrdoob/three.js/tree/master/editor/js/commands
We probably need to hook our events componentadd, componentremove, entityupdate, entityidchange, objectremove, entitycreated, entityclone, etc with a system like that.

**Comment by vincentfretin** [comment]
That feature has been implemented in aframe-editor in https://github.com/c-frame/aframe-editor/pull/6
